share/template.html

```diff
@@ -201,31 +208,105 @@
                 }
 
                 function mouseover() {
-                  d3.select(this).select("circle").transition()
-                      .duration(500)
-                      .attr("r", function(d){
-                        return cRadius * 1.1;
-                      })
-                      .style("fill", function(d) {
-                         return "#90EE90";
-                      })
-                  d3.select(this).select('text').transition()
-                      .duration(500)
-                      .style("font-size", function(d) { return d.scale * 1.1; });
+                  if (nodeSelected === null) {
+                    d3.select(this).select("circle").transition()
+                        .duration(500)
```

Comment:
haven't tested this out yet but I was having some weird issues with transitions for the fade in/out. I ended up having to just skip the transition altogether. this may not apply here